flash

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+689#!/usr/bin/env bash
```

Comment:
this looks not correct

Reply:
You're right - Added that by accident... Fixing now.